tests/reference/asr-array_01_decl-39cf894.stderr

```diff
@@ -0,0 +1,17 @@
+warning: The module 'numpy' located in $DIR/src/bin/../runtime/lpython_intrinsic_numpy.py cannot be loaded
+ --> tests/../integration_tests/array_01_decl.py:2:1
+  |
+2 | from numpy import empty, int16, int32, int64, float32, float64, complex64, complex128
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ imported here
+
```

Comment:
It seems there are several errors like above that got committed in this PR. Are these reference errors expected or is it a bug?